task/gcs-download/0.1/gcs-download.yaml

```diff
@@ -55,7 +55,7 @@ spec:
 
       if [[ "$(params.typeDir)" == "true" ]]; then
         mkdir -p "$DESTINATION"
-        gsutil rsync -d -r "$(params.location)" "$DESTINATION"
+        gsutil -m rsync -d -r "$(params.location)" "$DESTINATION"
```

Comment:
Is there drawback to multiplex the connexions? Should it be optional?

Reply:
Yes-ish; from `gsutil help options`:

Using the -m option may make your performance worse if you
are using a slower network, such as the typical network speeds
offered by non-business home network plans. It can also make
your performance worse for cases that perform all operations
locally (e.g., gsutil rsync, where both source and destination
URLs are on the local disk), because it can "thrash" your local
disk.

The latter case does not apply here. The former case if a possibility, but even on my slow home American "broadband" connection I see improved performance.

Reply:
Sounds good to me! thanks, that doesn't sound that could happen in our case (i.e from a task)